flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/CheckpointBarrierAlignerTestBase.java

```diff
@@ -488,47 +288,50 @@ public void testMultiChannelSkippingCheckpoints() throws Exception {
 
 		// align checkpoint 1
 		startTs = System.nanoTime();
+		check(sequence[5], inputGate.pollNext().get(), PAGE_SIZE);
+		toNotify.setNextExpectedCheckpointId(1);
```

Comment:
why add this `toNotify.setNextExpectedCheckpointId(1)`, i guess it is not needed